release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -50,6 +50,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-13-installation-vsphere-8-support"]
+==== Support for VMware vSphere version 8.0 
+{product-title} {product-version} supports VMware vSphere version 8.0. You can continue to install an {product-title} cluster on VMware vSphere version 7.0 Update 2.
+
+[id="ocp-4-13-installation-vsphere-region-zone"]
+==== VMware vSphere region and zone enablement
+You can deploy an {product-title} cluster to multiple vSphere datacenters, or regions, that run in a single VMware vCenter. Each datacenter can run multiple clusters, or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
```

Comment:
I am not sure the commas are needed in the first sentence. I know the comma in the second sentence is unnecessary. 

```suggestion
You can deploy an {product-title} cluster to multiple vSphere datacenters or regions that run in a single VMware vCenter. Each datacenter can run multiple clusters or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
```

Reply:
Updated